### ISSUE TITLE
Different result?

### ISSUE THREAD

**Issue by evi-Genius** [DingXiaoH/ACNet#1]
Hi. I have downloaded your github code and trained the ResNet-56 on cifar10. But the result is totally different from your paper.
In the paper, ResNet-56  trained on cifar10:
Model Base Top-1 ACNet Top-1  
ResNet-56            94.31    95.09

But in my experiment,
> python acnet_rc56.py --try_arg=normal_lrs1

INFO validate at epoch 496, top1=93.80000, top5=99.80000, loss=0.320561.80000, top5=99.80000, loss=0.32056

> python acnet_rc56.py --try_arg=acnet_lrs1

validate at epoch 492, top1=91.70000, top5=99.75000, loss=0.442718.70000, top5=99.75000, loss=0.44272

My pytorch version:
torch==1.2.0
torchvision==0.4.0

You would see in my experiment the ACNet result is worse than normal training and I have done it twice. The results have no change.

**Comment by DingXiaoH** [comment]
I am checking. Will see the results tomorrow.

**Comment by DingXiaoH** [comment]
Just found the bugs. I forgot to handle the 'padding' argument of res block with stride=2. I am testing.

**Comment by evi-Genius** [comment]
> Just found the bugs. I forgot to handle the 'padding' argument of res block with stride=2. I am testing.


> Just found the bugs. I forgot to handle the 'padding' argument of res block with stride=2. I am testing.

I see... Please tell me the result when it comes out :D

**Comment by DingXiaoH** [comment]
Bug fixed. It turns out that I forgot to add 'ReLU' in 'Conv2dBNReLU' of acnet_builder. Thank you for being interested in this paper.
